docs/src/userguide/anatomy/graph.rst

```diff
@@ -1,9 +1,19 @@
 Graph
 =====
 
-In **streaming** mode, Klio makes use of `Google Pub/Sub`_ and `GCS buckets`_ to create a directed acyclic graph (DAG) to string job dependencies together, allowing various modes of execution.
 
-Klio support two modes of execution: :ref:`top-down <top-down>` and :ref:`bottom-up <bottom-up>`.
+Management of a graph of Klio jobs is not supported in **batch** mode.
+We recommend making use of an external orchestration framework (such as `Luigi`_)
+to manage dependencies between Klio batch jobs. In addition, you may find using your job's
+:ref:`event outputs <io-outputs>` helpful.
```

Comment:
Curious, how would this be helpful in this case?

Reply:
Hmm, is this too vague to be useful?

I was thinking that downstream batch pipelines might check the event output table/file to determine the entities they needed to process.

Reply:
Ah I see. Maybe clarify saying something that Klio batch isn't able to trigger parent (batch) jobs like it can in streaming for non-existent input data. But you may still use Klio's (default or custom) data input existence checks to drop messages where input data doesn't exist, as well as data output existence checks to skip already-processed items.

Reply:
Updated! And thanks - I borrowed quite a bit of your phrasing.